app/jobs/update_diaper_partner_job.rb

```diff
@@ -6,7 +6,11 @@ class UpdateDiaperPartnerJob
   def perform(partner_id)
     @partner = Partner.find(partner_id)
     @invitation_message = @partner.organization.invitation_text
-    @response = DiaperPartnerClient.post(@partner.attributes, @invitation_message) if Flipper.enabled?(:email_active)
+    @response = DiaperPartnerClient.post(partner_attributes(@partner), @invitation_message) if Flipper.enabled?(:email_active)
     @partner.invited!
   end
+
+  def partner_attributes(partner)
+    partner.attributes.merge({ organization_email: partner.organization.email }).with_indifferent_access
+  end
```

Comment:
We considered doing it as a private method -- I agree that from a design perspective that makes sense; we left it public so it could be tested directly. (Testing private methods with RSpec is tricky, as you know)

I like your spec suggestion below, though :+1: